tests/zfs-tests/tests/functional/cli_root/zfs_diff/zfs_diff_cliargs.ksh

```diff
@@ -39,7 +39,7 @@ function cleanup
 log_assert "'zfs diff' should only work with supported options."
 log_onexit cleanup
 
-typeset goodopts=("" "-F" "-H" "-t" "-FH" "-Ft" "-Ht" "-FHt")
+typeset goodopts=("" "-p" "-t" "-tp" "-H" "-Hp" "-Ht" "-Htp" "-F" "-Fp" "-Ft" "-Ftp" "-FH" "-FHp" "-FHt" "-FHtp")
```

Comment:
I believe you missed updating this, unless I'm inverting what good and bad mean here.

Reply:
i did!